### ISSUE TITLE
Suggestion for Task's module - For migrating Users from other CRM

### ISSUE THREAD

**Issue by GrupImatica** [salesagility/SuiteCRM#2567]
We are a SAGE Distributor, and we are planning to migrate SAGE user's to SUITE CRM, but i share some inputs that have SAGE CRM that make's some tasks more efficient.

1.- When you create a TASK, in SAGE CRM you have a button to create a new TASK related, this one copies Automaticlly SUBJECT to a new task, and  put focus on DUE date to set time limit.

With one clic, you make a completed task and created a new other task Pending

2.- Possibility to have a TEAM and asign task to a TEAM, and then when user give's the task AutoAssign to their user.

I don't Know if is good place to share this here. But, i see in forum there are no active activity, and here i see that are a lot of contrinutors actives.

Thanks for your support, and we are very happy to work with SuiteCRM.

Thanks a lot.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Suggestion for Task's module - For migrating Users from other CRM](https://trello.com/c/DIlbUY4u/57-suggestion-for-task-s-module-for-migrating-users-from-other-crm)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
